docs/services/slam/cartographer/_index.md

```diff
@@ -214,46 +214,28 @@ Select the **Raw JSON** mode, then copy/paste the following `"services"` and `"m
 
 ### Adjust `data_dir`
 
-Change the `data_dir` attribute to point to a directory on your machine where you want to save the sensor data your SLAM service uses and the maps and config files it produces.
+Change the `data_dir` attribute to point to a directory on your machine where you want to save the internal state your SLAM service produces.
 
 This directory must be structured as follows:
 
 <pre>
 .
 └──\(<file>CARTOGRAPHER_DIR</file>)
-    ├── <file>map</file>
-    ├── <file>data</file>
-    └── <file>config</file>
+    ├── <file>internal_state</file>
 </pre>
 
 Click through the following tabs to see the usage of each folder in this directory:
 
-{{% tabs name="Folders"%}}
-{{% tab name="/map" %}}
-
-Whether mapping data is present in <file>map</file> at runtime and the attribute `map_rate_sec` determines the SLAM mapping mode:
+Whether internal state data is present in <file>internal_state</file> at runtime and the attribute `map_rate_sec` determines the SLAM mapping mode:
```

Comment:
I think this no longer makes sense in the context of the docs
<img width="758" alt="Screenshot 2023-07-27 at 3 41 37 PM" src="https://github.com/viamrobotics/docs/assets/121991867/ec9f17da-fea1-43fc-903d-432e233b91cb">